### PR TITLE
Added support of fixed distance binary sensors & adheres now the range message specification

### PR DESCRIPTION
**Pull request by AlexReimann**
Also changed / updated project file and formatted to ROS format

### CODE REVIEW

**Comment by AlexReimann** [comment]
@bit-pirate

File: range_sensor_layer/src/range_sensor_layer.cpp [hunk 1]
```diff
@@ -82,54 +84,53 @@ void RangeSensorLayer::onInitialize()
...
 }
 
-
 double RangeSensorLayer::gamma(double theta)
 {
-    if(fabs(theta)>max_angle_)
-        return 0.0;
-    else
-        return 1 - pow(theta/max_angle_, 2);
+  if (fabs(theta) > max_angle_)
```
Comment (bit-pirate) [inline range_sensor_layer/src/range_sensor_layer.cpp:89]: The ROS format should bring in brackets here. Please check.

**Comment by bit-pirate** [comment]
Two comments:

1) For PRs it's general a good idea to collapse multiple commits for the same feature/bug/etc. into one. This helps to reduce the "commit noise", git rebase is the key here. I can give you an introduction tomorrow.

2) Let's do a quick functional test tomorrow after which I'll merge this.

**Comment by bit-pirate** [comment]
Test successful.

**Pull request merged**